src/Stryker.Core/Stryker.Core/Mutators/CheckedMutator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.CodeAnalysis.CSharp.Syntax;
+using System.Collections.Generic;
+using Stryker.Core.Mutants;
+using Microsoft.CodeAnalysis.CSharp;
+
+namespace Stryker.Core.Mutators
+{
+    public class CheckedMutator : Mutator<CheckedExpressionSyntax>, IMutator
+    {
+        public override IEnumerable<Mutation> ApplyMutations(CheckedExpressionSyntax node)
+        {
+            if (node.Kind() == SyntaxKind.CheckedExpression)
+            {
+                yield return new Mutation()
+                {
+                    OriginalNode = node,
+                    ReplacementNode = node.Expression,
+                    DisplayName = "Checked expression to expression mutation",
```

Comment:
I think this would look better like: "Remove checked expression". What do you think?

Reply:
Good idea!